contentgrid-appserver-rest/src/main/java/com/contentgrid/appserver/rest/exception/EmptyRelationException.java

```diff
@@ -0,0 +1,11 @@
+package com.contentgrid.appserver.rest.exception;
+
+import com.contentgrid.appserver.domain.values.RelationRequest;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class EmptyRelationException extends RuntimeException {
+    private final RelationRequest relationRequest;
```

Comment:
This should probably be a `RelationIdentity` instead (or `EntityIdentity` + `RelationName`), as a `RelationRequest` is a request for a relation, not the identification of a certain relation.